### PR TITLE
LPS-199427 Scheduled articles not exported

### PR DESCRIPTION
**Pull request by adolfopa**
# What is this trying to solve?

https://liferay.atlassian.net/browse/LPS-199427

Only approved articles are exported from staged sites.

# How am I fixing it?

I simply added the scheduled status to the set of exportable statuses.

# How can you verify that it works?

Follow the steps described in https://liferay.atlassian.net/browse/LPS-199427.

### CODE REVIEW

**Review by manuelRives — APPROVED** [review]

File: modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/exportimport/data/handler/KBArticleStagedModelDataHandler.java [hunk 1]
```diff
@@ -102,6 +102,14 @@ public String getDisplayName(KBArticle kbArticle) {
 		return kbArticle.getTitle();
 	}
 
+	@Override
+	public int[] getExportableStatuses() {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/exportimport/data/handler/KBArticleStagedModelDataHandler.java:106]: Hi @adolfopa 

I tried this but it fail me the second time I was trying to publish the site to live.

Can you check it if it fails to you too?

Comment (manuelRives) [reply modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/exportimport/data/handler/KBArticleStagedModelDataHandler.java:106]: I also got an error when editing and republishing it.

image

**Comment by AliciaGarciaGarcia** [comment]
Hi @adolfopa 

I reviewed your PR and seems to be ok

But I found an issue with the elements shown on [CHANGES SINCE LAST PUBLISH PROCESS] section that does not register the previous published changes.

It's not caused by your PR. I tried without your changes.

https://github.com/liferay-lima/liferay-portal/assets/47524751/47b8f368-a365-4a6a-93c5-12cb7b2217c9

I you want to take a look at it and see if is doable on this PR, it's ok.
And if you do this. a test for the BaseStagedModelDataHandler will be welcome too ;)

**But this PR as standalone is perfect.**

I let you decide forward it.

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**